hieradata/node/lsstcam-rebps.cp.lsst.org.yaml

```diff
@@ -0,0 +1,32 @@
+---
+nm::connections:
+  enp0s31f6:
+    content:
+      connection:
+        id: "enp0s31f6"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "enp0s31f6"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.78/26,139.229.175.126"
+        dns: "139.229.160.53;139.229.160.54;139.229.160.55;"
+        dns-search: "cp.lsst.org;"
+        method: "manual"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  enp3s0:
+    content:
+      connection:
+        id: "enp3s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp3s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.127/24"
```

Comment:
Is this directly connected to another device or is there a switch and multiple devices?  If so, I think we should put the lsstcam private network(s) into ipam.

Reply:
I think it will be eventually connected to a private network. But this will be defined later by Stuart Marshall and Tony. Once the private network is set and the IP that this node will have i will update this IP and will add the private subnet to IPAM but nothing is set in stone from what aware off.

Reply:
So per Stuarts comments on slack. These hosts will live under the following private subnet: 192.168.1.0/24 and will have the following IPs:

- lsstcam-rebps  192.168.1.121
- lsstcam-fcs   192.168.1.123
- lsstcam-quad   192.168.1.120
- lsstcam-vac   192.168.1.122

Im going to create the private network in IPAM to start populating it. 

These connections will got to an unmanaged industrial switch that will handle communication in a private network between the different camera components (i/o modules, plcs. etc..).